maxi-blocks/maxi-blocks
# Issue 2474 - ColorControl reset button

I have dealt with the comments you gave me. Check it out and ping me back.

-------------------------------------------------------
@Naaaaiix comment:
To test:
1. On changing a palette colour, when reseting it should give back the default palette colour or the responsive previous one.
2. On change from palette to custom, custom should get the palette colour
3. On change custom colour and then reset, custom colour should come back to the palette colour default (not changing to palette selector)

Note: BackgroundColor layers is not perfectly working, needs to be fixed. As is part of something bigger, will be derive to a new issue onces this PR will be merged 👍
